charts/traefik/v1.7/questions.yml

```diff
@@ -0,0 +1,118 @@
+categories:
+- Proxy
+- Loadbalancer
```

Comment:
can we please add the following label, will need to for our multi-cluster app filtering, thanks.
```
labels:
  io.cattle.role: project # options are cluster/project
```

Reply:
Done :)